1310-x86-smpboot.c-TXT-AP-bringup.patch

```diff
@@ -0,0 +1,177 @@
+From 23804dc59d572ccd5e1d61afa9f24111da4f3ace Mon Sep 17 00:00:00 2001
+From: Krystian Hebel <krystian.hebel@3mdeb.com>
+Date: Wed, 16 Nov 2022 15:06:18 +0100
+Subject: [PATCH 10/11] x86/smpboot.c: TXT AP bringup
+
+On Intel TXT, APs are started in one of two ways, depending on ACM
+which reports it in its information table. In both cases, all APs are
+started simultaneously after BSP requests them to do so. Two possible
+ways are:
+- GETSEC[WAKEUP] instruction,
+- MONITOR address.
+
+This patch implements just the latter, GETSEC[WAKEUP] support will be
+added later.
```

Comment:
It is hard to find any concrete information on this. It requires versions >= 7 of SINIT to MLE Data, but there is no clear mapping of that version with regard to processor family. I haven't seen any CPU that would actually use it. I can imagine it was designed for TXT support for CPUs that lack MONITOR/MWAIT, because `GETSEC[WAKEUP]` seems to be more complicated, in software and hardware alike.

I don't feel comfortable subbmitting code that wasn't tested, and we don't have a way of testing it as of now.